src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -115,8 +115,20 @@ function iterator(array $values): IteratorArgument
 
 /**
  * Creates a lazy iterator by tag name.
+ *
+ * @deprecated since version 4.4, to be removed in 5.0. Use "tagged_iterator" instead.
```

Comment:
since Symfony [...]

Reply:
I just copied this from https://symfony.com/doc/current/contributing/code/conventions.html#deprecations But sure I can change it. Does it make sense to adjust the documentation?